src/providers/transactionManager/useTransaction.ts

```diff
@@ -57,7 +55,11 @@ export const useTransaction = (): HandleTransactionFn => {
       if (preProcess) {
         setDialogStep(ExtrinsicStatus.ProcessingAssets)
         try {
-          await preProcess()
+          await new Promise((resolve) =>
+            setTimeout(() => {
+              resolve(preProcess())
+            }, 2000)
```

Comment:
If I'm not mistaken this change is supposed to make the first step displayed even if there's no processing to do, right? This is the UI/design leaking into the transaction handling logic which we should avoid. This has to be somehow handled by the dialog itself.

Reply:
Exactly. This is possible to handle it by the dialog but this way Polkadot extension opens while you have `Processing your assets` dialog open.

Reply:
Additionally, maybe instead of predefined value of 2s we just show the animation for one loop?

Reply:
Oh yes I see the issue

Reply:
Based on the Rocket chat discussion we don't need this